File: C programming/Assignment/Assignment5/type3/firstandlast.c
#include <stdio.h>

void operation(int*);
void main(){
    int num;
    printf("Enter the number:");
    scanf("%d",&num);
    operation(num);
}
void operation(int num){
    int last,first,sum,temp;

    temp = num;
    last = num % 10;
    
    while(temp>10){
        temp = temp / 10;
    }

    first = temp;
    sum = first+last;
    printf("Addition of first and last number is %d",sum);
}

File: Java_programming/basic_java/assignment3/firstandlast.java
package assignment3;

public class firstandlast {

	public static void main(String[] args) {
		int num=1234,last,first,sum,temp;
//	    printf("Enter the number:");
//	    scanf("%d",&num);

	    temp = num;
	    last = num % 10;
	    
	    while(temp>10){
	        temp = temp / 10;
	    }

	    first = temp;
	    sum = first+last;
	    System.out.printf("Addition of first and last number is %d",sum);

	}

}
